docs/reference/release-notes/6.7.0.asciidoc

```diff
@@ -0,0 +1,515 @@
+
+Security::
+* Deprecate implicit security on trial licenses {pull}38295[#38295] (issues: {issue}38009[#38009], {issue}38075[#38075])
+
+
+
+[[feature-6.7.0]]
+[float]
+=== New features
+
+Authentication::
+* Add support for API keys to access Elasticsearch {pull}38291[#38291] (issue: {issue}34383[#34383])
+
+Authorization::
+* Allow custom authorization with an authorization engine  {pull}38358[#38358] (issues: {issue}32435[#32435], {issue}36245[#36245], {issue}37328[#37328], {issue}37495[#37495], {issue}37785[#37785], {issue}38137[#38137], {issue}38219[#38219])
+* WIldcard IndicesPermissions don't cover .security {pull}36765[#36765]
+
+CCR::
+* Add ccr follow info api {pull}37408[#37408] (issue: {issue}37127[#37127])
+
+Features/ILM::
+* [ILM] Add unfollow action {pull}36970[#36970] (issue: {issue}34648[#34648])
+
+Features/Ingest::
+* Enable grok processor to support long, double and boolean {pull}27896[#27896]
+
+Machine Learning::
+* ML: Adds set_upgrade_mode API endpoint {pull}37837[#37837]
+
+Mapping::
+* Give precedence to index creation when mixing typed templates with typeless index creation and vice-versa. {pull}37871[#37871] (issue: {issue}37773[#37773])
+* Add an `include_type_name` option to 6.x. (#29453) {pull}37147[#37147] (issue: {issue}35190[#35190])
+
+SQL::
+* SQL: Allow sorting of groups by aggregates {pull}38042[#38042] (issue: {issue}35118[#35118])
+* SQL: Implement FIRST/LAST aggregate functions {pull}37936[#37936] (issue: {issue}35639[#35639])
+* SQL: Introduce SQL DATE data type {pull}37693[#37693] (issue: {issue}37340[#37340])
+
+
+
+[[enhancement-6.7.0]]
+[float]
+=== Enhancements
+
+Aggregations::
+* Add Composite to AggregationBuilders {pull}38207[#38207] (issue: {issue}38020[#38020])
+* Allow nested fields in the composite aggregation {pull}37178[#37178] (issue: {issue}28611[#28611])
+* Remove single shard optimization when suggesting shard_size {pull}37041[#37041] (issue: {issue}32125[#32125])
+* Use List instead of priority queue for stable sorting in bucket sort aggregator {pull}36748[#36748] (issue: {issue}36322[#36322])
+* Keys are compared in BucketSortPipelineAggregation so making key type… {pull}36407[#36407]
+
+Audit::
+* Security Audit includes HTTP method for requests {pull}37322[#37322] (issue: {issue}29765[#29765])
+* Add X-Forwarded-For to the logfile audit {pull}36427[#36427]
+
+Authentication::
+* Security: propagate auth result to listeners {pull}36900[#36900] (issue: {issue}30794[#30794])
+* Security: reorder realms based on last success {pull}36878[#36878]
+* Deprecation check for Auth realm setting structure {pull}36664[#36664] (issue: {issue}36024[#36024])
+
+Authorization::
+* Permission for restricted indices {pull}37577[#37577] (issue: {issue}34454[#34454])
+* Create snapshot role {pull}35820[#35820] (issue: {issue}34454[#34454])
+
+CCR::
+* Integrate retention leases to recovery from remote [OPEN] {pull}38829[#38829] (issue: {issue}37165[#37165])
```

Comment:
We need to fix the labels on this one as the PR is showing as still open. We should label it as 6.7.1 for now and then the relabel process will move it back to 7.0.0 when a build candidate contains this change after its merged

Reply:
Oh, I missed that this one is still open. Thanks for spotting.

Reply:
As discussed offline, we expect that this particular PR will be included in 6.7.0 and thus I will not relabel but only remove it from the release notes for now.